### ISSUE TITLE
Screensaver is not notarized

### ISSUE THREAD

**Issue by lucpotage** [Wandmalfarbe/Simple-Clock-Screensaver#2]
Hello! I tried to installed this screensaver but with macOS Catalina, it seems screensavers must now be notarized. Do you think you could solve this issue?

Thanks.

**Comment by Chuckable-Nerd** [comment]
Hi @lucpotage ,

I am not sure if you are still looking to use this screensaver or not but you are able to get around the signature requirement by opening Finder and navigating to Macintosh HD/Library/Screen Savers and then Control-clicking on "SimpleClock.saver" (or just "SimpleClock" depending on your Finder settings) and clicking open. Now if you go into system preferences and open the screen saver settings you should now be able to click on SimpleClock and be able to "open" which will allow the screensaver to be used! 

Hope this helped, let me know if you run into any trouble!
Chuckable-Nerd

**Comment by lucpotage** [comment]
Hello @Chuckable-Nerd,

I tested what you describe and now I have a warning saying the developer is not verified. I only have two options:

- Move to trash
- Cancel

**Comment by Chuckable-Nerd** [comment]
Did you Control Click or right click (if enabled) and click open on Simple Clock? Doing this should give you a third option to open

**Comment by lucpotage** [comment]
Yes but it does not work. Rules may be even stricter with macOS 11 Beta?

**Comment by Chuckable-Nerd** [comment]
Potentially. Maybe try looking in system preferences, security and privacy, general to see if it says anything about the program being blocked?

This will probably come up when you try running SimpleClock.saver

**Comment by BlinksTale** [comment]
Sadly none of these seem to work currently, even in the privacy and security settings (it doesn't appear there). Would love to see a notarized version of this app, but for now I see all the same issues as lucpotage.

EDIT: I spoke too soon! By clicking a different screensaver, then clicking back to this one, Settings successfully refreshed. This was with the `.saver` file in `/Library/Screen Savers/` I believe.

> macOS cannot verify the developer of “SimpleClock.saver”. Are you sure you want to open it?
> ...
> Open
> Show in Finder
> Cancel

Clicking "Open" from here gets it to successfully work. Thank you!

**Comment by BluefusionUK** [comment]
I've followed all the above comments, I can now get past all the warnings, however, when I preview or run the screensaver my screen is just black? Running OSX 15.4.1
